README.markdown

```diff
@@ -23,6 +23,17 @@ inherit_gem:
     - default.yml
 ```
 
+## Updating Gem
```

Comment:
Can this be automated through github actions?

Reply:
Yes, we can definitely do that eventually.